style.css

```diff
@@ -124,3 +136,255 @@ p {
   text-align: center;
   margin-top: 65px;
 }
+
+.projects {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.head {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 42px;
+  color: #172b4d;
+  padding: 24px;
+}
+
+.span {
+  width: 48px;
+  border: 3px solid #36b37e;
+}
+
+.container {
+  display: grid;
+  grid-template-rows: auto auto auto auto auto auto;
+  gap: 24px;
+  padding: 62px 24px;
+}
+
+.card {
+  height: 474px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  background: #ebf0ee;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+.title {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  border-radius: 8px;
+  padding-bottom: 24px;
+  gap: 8px;
+  height: 222px;
+}
+
+.text {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #3a4a42;
+}
+
+.server {
+  display: flex;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+}
+
+.server li {
+  padding: 8px 12px;
+  background: #ebf0ee;
+  border-radius: 4px;
+  list-style: none;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  color: #3a4a42;
+}
+
+.button {
```

Comment:
Kindly update your button element to include button interaction styling when `hovered ` and `pressed`🛠️. You can use `.button:hover` and `.button:active` to style such interactions. Please ensure to match the design guide provided by figma 🎨

Figma design:
<img width="271" alt="image" src="https://user-images.githubusercontent.com/37116322/219165111-37148da3-467f-40bb-a9c3-bef34f5384bf.png">

Reply:
- [ ] As the previous reviewer suggested, please implement the button states `hover` & `pressed` 🔧

Reply:
Could you please recheck, the review was done on the wrong branch🙂